cfme/tests/cloud/test_cloud_proxy.py

```diff
@@ -0,0 +1,153 @@
+import pytest
+
+import cfme.fixtures.pytest_selenium as sel
+from cfme import test_requirements
+from cfme.cloud.provider import CloudProvider
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.web_ui import flash, form_buttons
+from utils import testgen, error
+from utils.conf import cfme_data, credentials
+from utils.log import logger
+from utils.appliance.implementations.ui import navigate_to
+
+pytestmark = [
+    pytest.mark.tier(1),
+    test_requirements.proxy,
+    pytest.mark.usefixtures('uses_cloud_providers')
```

Comment:
I removed setup_provider because of another reviewer, so I'm kind of stuck here.